whatwg-streams/index.d.ts

```diff
@@ -18,8 +18,8 @@ interface ReadableByteStreamSource {
 }
 
 interface QueuingStrategy {
-    highWaterMark?: number;
```

Comment:
Follow the same declaration order as in the spec: easier when reading the spec, see https://streams.spec.whatwg.org/#rs-class-definition